Fix error message lost issue (#6057)
dapr/dapr: docs/release_notes/v1.10.3.md

### Solution

We disabled the redirect handle and returned the original response to the caller.

## Fixed error message lost

### Problem

> Application using GRPC protocol returns an error message, the error message is lost and replaced with `message is nil`.

Reviewer: ```suggestion
When Dapr invokes an application using the gRPC protocol that returns an error message, the returned error message is lost and replaced with `message is nil`.
```

Author: Done, thanks for your detailed review.

### Impact

> This impacts users on Dapr 1.10.0-1.10.2, application using GRPC protocol and the endpoint returns an error message.

Reviewer: ```suggestion
This impacts users on Dapr 1.10.0-1.10.2 that invoke applications using Dapr and the gRPC protocol.
```

Author: Done
